test-poc/framework/src/main/java/org/keycloak/test/framework/webdriver/ChromeWebDriverSupplier.java

```diff
@@ -0,0 +1,42 @@
+package org.keycloak.test.framework.webdriver;
+
+import org.keycloak.test.framework.injection.InstanceWrapper;
+import org.keycloak.test.framework.injection.LifeCycle;
+import org.keycloak.test.framework.injection.Registry;
+import org.keycloak.test.framework.injection.Supplier;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class ChromeWebDriverSupplier implements Supplier<WebDriver, TestWebDriver> {
+
+    @Override
+    public Class<TestWebDriver> getAnnotationClass() {
+        return TestWebDriver.class;
+    }
+
+    @Override
+    public Class<WebDriver> getValueType() {
+        return WebDriver.class;
+    }
+
+    @Override
+    public InstanceWrapper<WebDriver, TestWebDriver> getValue(Registry registry, TestWebDriver annotation) {
+        ChromeDriver driver = new ChromeDriver();
+        return new InstanceWrapper<>(this, annotation, driver);
+    }
+
+    @Override
+    public LifeCycle getLifeCycle() {
+        return LifeCycle.GLOBAL;
+    }
+
+    @Override
+    public boolean compatible(InstanceWrapper<WebDriver, TestWebDriver> a, InstanceWrapper<WebDriver, TestWebDriver> b) {
+        return a.getValue().getClass().equals(b.getValue().getClass());
```

Comment:
Would this statement not always be `true` in the current system? I think we might just be able to get away with hard-coding this for now.  
```suggestion
        return true;
```

Reply:
This is correct. It can be set to `true` for now